packages/plugin-messagebroker/test/MessageBrokerPlugin.test.ts

```
@@ -0,0 +1,219 @@
+        // @ts-ignore
+        plugin['transportSubscriptions'].set(subId, { type: 'rabbitmq', queueOrChannelName: testQueue, rabbitConsumerTag: 'tag1', specificHandler: handler });
+        await plugin.close(subId);
+        expect(mockRabbitMQServiceInstance.unsubscribe).toHaveBeenCalledWith(testQueue, handler, 'tag1');
+    });
+  });
+
+  describe('ITransport Implementation - Redis', () => {
+    const testChannel = 'test-chan';
+    const testMessage: Message = 'redis message';
+    let handler: jest.Mock<MessageHandler>;
+    let handler: jest.Mock<MessageHandler>;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handler'. Consider to delete it or rename it.

'handler' is defined here:


(lint/suspicious/noRedeclare)